static/app/stores/groupingStore.tsx

```diff
@@ -495,7 +495,8 @@ const storeConfig: GroupingStoreDefinition = {
           });
           this.unmergeList.clear();
         },
-        error: () => {
+        error: error => {
+          errorMessage = error?.responseJSON?.detail || errorMessage;
```

Comment:
any reason to not use `??` instead of `||`

Reply:
@roggenkemper If by chance it's falsy then this is better